### PR TITLE
Add support for Zoom display name in profiles

### PR DESCRIPTION
**Pull request by skddc**
Currently depends on an unreleased version of `kredits-contracts` (which also fixes the broken profile updates).

### CODE REVIEW

File: app/controllers/contributors/new.js [hunk 1]
```diff
@@ -8,9 +8,7 @@ export default Controller.extend({
   actions: {
 
     save (contributor) {
-      return this.kredits
-                 .addContributor(contributor)
-                 .then(() => this.transitionToRoute('index'))
```
Comment (galfert) [inline app/controllers/contributors/new.js]: Does this stay on the form page now, or is the transition happening somewhere else when editing or creating a new contibutor?

Comment (skddc) [reply app/controllers/contributors/new.js]: Yes, I moved it to the component:

https://github.com/67P/kredits-web/pull/156/files#diff-05b2a32d1311a3737ec444f68955ccedR81

**Comment by bumi** [comment]
I've updated the package.json to use the released kredits-contracts v5.5.0
but if i do an `npm install` there are loads of changes in the package-lock file (some lodash changes, camelcase version change, etc.) which I did not want to push. 
Could you push an updated package-lock?

**Comment by skddc** [comment]
@bumi OK, cool. I just ran `npm install` and only got the `kredits-contracts` change.

**Pull request merged**